langkit/libmanage.py

```diff
@@ -257,6 +257,11 @@ def internal(*args, **kwargs):
             help='Installation directory.'
         )
 
+        install_parser.add_argument(
+            '--force', '-f', action='store_true', dest='force', default=False,
```

Comment:
Both `dest` and `default` arguments can be removed (`default=False` is implied by `action='store_true'` and `dest='force'` is implied by `--force`).

Reply:
ok, good, i'll fix

Reply:
Fixed

Reply:
Thanks!